plugins/editors/tinymce/src/PluginTraits/DisplayTrait.php

```diff
@@ -377,7 +377,7 @@ public function onDisplay(
 		{
 			$externalPlugins['jdragndrop'] = HTMLHelper::_('script', 'plg_editors_tinymce/plugins/dragdrop/plugin.min.js', ['relative' => true, 'version' => 'auto', 'pathOnly' => true]);
 			$uploadUrl                     = Uri::base(false) . 'index.php?option=com_media&format=json&url=1&task=api.files';
-			$uploadUrl                     = $this->app->isClient('site') ? htmlentities($uploadUrl, null, 'UTF-8', null) : $uploadUrl;
+			$uploadUrl                     = $this->app->isClient('site') ? htmlentities($uploadUrl, 0, 'UTF-8', false) : $uploadUrl;
```

Comment:
Please use the PHP Constant that was designed for this purpose `ENT_NOQUOTES` - see https://3v4l.org/70HcC

```suggestion
			$uploadUrl                     = $this->app->isClient('site') ? htmlentities($uploadUrl, ENT_NOQUOTES, 'UTF-8', false) : $uploadUrl;
```

Reply:
:+1: Good tip, doesn't change the value of 0 but improves the readability. Merging, Thank you, @PhilETaylor